### PR TITLE
Refactor theme primary color, Edit button position and fix popup content styling  

### PR DESCRIPTION
**Pull request by srsharon**
## Purpose
image
image

### CODE REVIEW

File: composer/modules/web/font/font-ballerina/icons/eclips.svg [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+ ~   Copyright 2017, WSO2 Inc. (http://www.wso2.com)
```
Comment (pahans) [inline composer/modules/web/font/font-ballerina/icons/eclips.svg]: year should be 2018

File: composer/modules/web/font/font-ballerina/icons/uneditable.svg [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+ ~   Copyright 2017, WSO2 Inc. (http://www.wso2.com)
```
Comment (pahans) [inline composer/modules/web/font/font-ballerina/icons/uneditable.svg]: year should be 2018

File: composer/modules/web/src/plugins/ballerina/views/definition-view-menu.jsx [hunk 3]
```diff
@@ -75,33 +75,42 @@ class DefinitionViewMenu extends React.Component {
...
-                flowing
-                hoverable
+                // flowing
+                // hoverable
                 wide
-                position='bottom center'
-                open={this.state.isOpen}
-                onClose={this.handleClose}
-                onOpen={this.handleOpen}
+                // position='bottom center'
```
Comment (pahans) [inline composer/modules/web/src/plugins/ballerina/views/definition-view-menu.jsx]: Shall we remove commented code?

File: composer/modules/web/font/font-ballerina/icons/eclips.svg (hunk 1 above)
Comment (srsharon) [reply composer/modules/web/font/font-ballerina/icons/eclips.svg]: Fixed

Comment (srsharon) [reply composer/modules/web/font/font-ballerina/icons/eclips.svg]: FIxed

**Pull request merged**